contrib/codegen/tests/ui-fail/database-types.rs

```diff
@@ -0,0 +1,10 @@
+extern crate rocket;
+#[macro_use] extern crate rocket_contrib;
+
+struct Unknown;
+
+#[database("foo")]
+struct A(Unknown);
+//~^ ERROR Unknown: rocket_contrib::databases::Poolable
+
```

Comment:
This would be a good place to add the regression test for #854, e.g. with `#[database("bar")] struct B(Vec<i32>);` -- that should error because `Vec<i32>` is not `Poolable`.